core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceMethod.java

```diff
@@ -467,10 +501,14 @@ ParameterType parameterType() {
             return parameterType;
         }
 
-        boolean isRequired() {
+        boolean required() {
             return required;
         }
 
+        public boolean optionalWrapped() {
```

Comment:
It is often better adding `is` for boolean properties because an adjective is sometimes a verb in English. e.g. empty

Reply:
Thanks! Fixed!